qiskit/circuit/library/boolean_logic/and_circuit.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
```

Comment:
this should just be and

Reply:
i tried that first but it conflicts with python keyword. it doesn't matter i think since the user just imports the class

Reply:
I realized after I wrote it. quantum_and, quantum_xor ?